consistent-sampling/build.gradle.kts

```diff
@@ -5,6 +5,7 @@ plugins {
 
 description = "Sampler and exporter implementations for consistent sampling"
 otelJava.moduleName.set("io.opentelemetry.contrib.sampler.consistent")
+otelJava.moduleName.set("io.opentelemetry.contrib.sampler.consistent2")
```

Comment:
A frivolous idea: use `consistent56` in the package name - to indicate the use of 56-bit randomness.

Reply:
resolved by 2e7cf7282e1f29fda401eb3ac7b2caa447b83be0